File: bigbang.py
import json

def generate_bigbang_array():
    result = []
    for i in range(1, 101):
        if i % 3 == 0 and i % 5 == 0:
            result.append("BIGBANG")
        elif i % 3 == 0:
            result.append("BIG")
        elif i % 5 == 0:
            result.append("BANG")
        else:
            result.append(str(i))
    return result

def save_to_json(data):
    # Write the array to output.json
    with open('output.json', 'w') as f:
        json.dump(data, f)

if __name__ == "__main__":
    # Generate the array
    array = generate_bigbang_array()
    save_to_json(array)
    print("Output has been saved to output.json") 

File: src/bigBang.ts
const fs = require('fs');

function generateBigBangArray(start: number, end: number): string[] {
    const result: string[] = [];
    
    for (let i = start; i <= end; i++) {
        if (i % 3 === 0 && i % 5 === 0) {
            result.push('BIGBANG');
        } else if (i % 3 === 0) {
            result.push('BIG');
        } else if (i % 5 === 0) {
            result.push('BANG');
        } else {
            result.push(i.toString());
        }
    }
    
    return result;
}

// Generate array from 1 to 100
const bigBangArray = generateBigBangArray(1, 100);

// Write to output.json
fs.writeFileSync('output.json', JSON.stringify(bigBangArray, null, 2));

console.log('Output has been written to output.json'); 